.editorconfig

```diff
@@ -14,3 +14,12 @@ trim_trailing_whitespace = true
 [*.json]
 indent_style = space
 indent_size = 2
+
+
+# ensure the LF line endings for SKTD markdowns
+[*.md]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
```

Comment:
This configuration must not run for all *.md files

Reply:
You mean, the `.md` file in the AFF for SKTD is allowed to have trailing whitespaces?

Reply:
Try adding to the original '.editorconfig'
```
[/file-formats/sktd/**/*.md]
trim_trailing_whitespace = unset
```

However, having trailing whitespaces might lead to undesired diffs.

Reply:
Thanks!

Yes, you are right. This is also what I mentioned earlier. See https://github.com/SAP/abap-file-formats/pull/270#issuecomment-992199027

Currently, @uenal-akkaya tends to keep them anyway. But this is still in discussion ;) If we can avoid this exemption, it would be great. :)